katdal/ms_async.py

```diff
@@ -0,0 +1,194 @@
+################################################################################
+# Copyright (c) 2011-2018, National Research Foundation (Square Kilometre Array)
```

Comment:
As something very minor, I usually pick the first year as the birth year of the specific module (i.e. `(c) 2018`).

Reply:
When I pressed copy that was my intention, but somehow by the time I pasted I'd forgotten. Fixed now.